tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Session.java

```diff
@@ -707,7 +707,7 @@ Graph graph() {
     return graph;
   }
 
-  private final Graph graph;
```

Comment:
Please @manojava98 , leave the field private and add a public getter named `graph()` to return that instance instead.